README.md

```diff
@@ -92,9 +92,16 @@ If the `aws_tag_select` feature was used, an additional information metric will
 # TYPE aws_resource_info gauge
 aws_resource_info{job="aws_elb",instance="",arn="arn:aws:elasticloadbalancing:eu-west-1:121212121212:loadbalancer/mylb",load_balancer_name="mylb",tag_Monitoring="enabled",tag_MyOtherKey="MyOtherValue",} 1.0
 ```
-
 All metrics are exported as gauges.
 
+The `aws_resource_info` can be used in conjunction with other metrics to expose tags using PromQL syntax like:
```

Comment:
This is out of place, as it's not directly following where aws_resource_info was mentioned. The grammar and terminology is also off. I'd say something like that aws_recource_info can be joined with other metrics using group_left in PromQL such as...